### ISSUE TITLE
Hotkey for decelerating camera movement is triggering Wii U's "Tap the touch screen" screen

### ISSUE THREAD

**Issue by mistermark** [etra0/botw-freecam#23]
When running a game for Wii U, and trying to decelerate the camera movement by pressing Tab, it will show the Wii U "Tap the touch screen" screen. The camera movement still works, but it's not possible to see where the camera is moving to due to the "Tap the touch screen" screen showing and blocking the view.

![obs-game-screenshot-2](https://user-images.githubusercontent.com/960311/112952691-11a52400-913d-11eb-8122-31158b1a9933.png)

**Comment by etra0** [comment]
Do you have any suggestion for a new key? I chose that one quite *randomly* since I use it with the controller more than the keyboard. I accept suggestions.

**Comment by IceWolf762** [comment]
> Do you have any suggestion for a new key? I chose that one quite _randomly_ since I use it with the controller more than the keyboard. I accept suggestions.

I'd like to suggest the Left CTRL key. It's right below SHIFT and would make it more convenient to use both SHIFT and CTRL for speed control.

**Comment by Duskangels** [comment]
Yes please! I'm having the same trouble. LOL. The right joystick turning speed it was too sensitive and the keyboard tab key is blocked by the Wii U message.

**Comment by WitherDragon6000** [comment]
In case this never got added, I found a workaround.

If you click of the terminal and give your inputs, then it will not activate the "toggle screen" function.
